app/views/bill-runs/review-licence.njk

```diff
@@ -83,23 +83,23 @@
   <form method="post">
     <div class="govuk-button-group">
     {{ govukButton({
-      text: statusButtonText,
+      html: '<div data-test="status-button">' + statusButtonText + '</div>',
```

Comment:
The button component allows you to add an `attributes` option which is where the `data-test` thingy should go.

Reply:
You don't need `data-test=` attributes for buttons! It is fine to use `text:` for those.

Checking out the [docs](https://docs.cypress.io/guides/references/best-practices#How-It-Works) this would mean we sit in ✅ **Depends** . But the button text so rarely changes it would be better to keep the views cleaner by just using the `text:` option.

Where `data-test=` really helps us is in summary lists and tables. Buttons, links and headings are normally very easy to select with cypress and also, very rarely change.